docs/gettingstarted/quickstart.ipynb

```diff
@@ -2,24 +2,175 @@
  "cells": [
```

Comment:
<p>Just one more cell above showing query functionality.</p><p> </p><p>For instance, Andy had </p>

<pre class="ql-syntax" spellcheck="false">all_sky_object = gaia.crossmatch(ztf).query(
    "nobs_g_ztf_dr14 &gt; 50 and nobs_r_ztf_dr14 &gt; 50 and \
    parallax_gaia &gt; 0 and parallax_over_error_gaia &gt; 5 and \
    teff_gspphot_gaia &gt; 5380 and teff_gspphot_gaia &lt; 7220 and logg_gspphot_gaia &gt; 4.5 \
    and logg_gspphot_gaia &lt; 4.72 and classprob_dsc_combmod_star_gaia &gt; 0.5")
</pre><p>something like that, just much simpler.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>